apps/vaporgui/PTFEditor.cpp

```diff
@@ -28,22 +29,24 @@ template class PTFMapWidget<TFIsoValueWidget>;
 
 PTFEditor::PTFEditor() : PTFEditor(RenderParams::_variableNameTag) {}
 
-PTFEditor::PTFEditor(const std::string &tag, const std::set<Element> elements, const std::string &label) : PWidget(tag, _section = new VSection(label.empty() ? tag : label))
+PTFEditor::PTFEditor(const std::string &tag, const std::set<Element> elements, const std::string &label, bool expandable) : PWidget(tag, _section = new VSection(label.empty() ? tag : label)), _expandable(expandable)
 {
     _maps = new TFMapGroupWidget;
     _histogram = new TFHistogramMap(tag);
     _opacityMap = new TFOpacityMap(tag);
     _colorMap = new TFColorMap(tag);
     _isoMap = new TFIsoValueMap(tag);
     _range = new TFMappingRangeSelector(tag);
+    _elements = elements;
+    _label = label;
```

Comment:
The label parameter should be held by the VSection rather than in the subclass. Also, it is optional which is not being handled.